Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php

```diff
@@ -32,6 +32,11 @@ public function testFixProvider()
         $template = '<?php %s';
         $tests = array();
         foreach (array('require', 'require_once', 'include', 'include_once') as $statement) {
+            $test[] = array(
+                sprintf('<?php %s "foo.php" ?>', $statement),
+                sprintf('<?php %s("foo.php") ?>', $statement),
```

Comment:
Sorry @keradus for not adding a description as I should have, it is indeed after the discussion as stated by @GrahamCampbell.